extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts

```diff
@@ -253,6 +253,11 @@ export class EncryptedMsgMailFormatter extends BaseMailFormatter {
     newMsgData: NewMsgData
   ): Promise<{ bodyWithReplyToken: SendableMsgBody; replyToken: string }> => {
     const recipients = getUniqueRecipientEmails(newMsgData.recipients);
+    for (const validRecipientElement of this.view.recipientsModule.getValidRecipients()) {
+      if (validRecipientElement.sendingType === 'bcc') {
+        recipients.splice(1, recipients.indexOf(validRecipientElement.email));
+      }
+    }
```

Comment:
let's make it simpler - we can remove bcc recipients before parsing unique emails:
```ts
const recipientsWithoutBcc = { ...newMsgData.recipients, bcc: [] }; // remove bcc recipients, so they're not included in reply data
const recipients = getUniqueRecipientEmails(newMsgData.recipients);
```

Reply:
thank you - I didn't know about this before. So basically, the `bcc` array from the de-sturured `newMsgData.recipients` object gets overridden by an empty array?

Reply:
Correct, so you can replace any object property with another value

Reply:
thank you very much. super helpful as always.